convert2rhel/systeminfo.py

```diff
@@ -99,10 +100,22 @@ def _get_system_name(self):
         return name
 
     def _get_system_version(self):
-        version = re.search(r".+?(\d+)\.?",
-                            self.system_release_file_content).group(1)
+        """Return a namedtuple with major and minor elements, both of an int type.
+        
+        Examples:
+        Oracle Linux Server release 6.10
+        Oracle Linux Server release 7.8
+        CentOS release 6.10 (Final)
+        CentOS Linux release 7.6.1810 (Core)
+        CentOS Linux release 8.1.1911 (Core)
+        """
+        match = re.search(r".+?(\d+)\.(\d+)\D?", self.system_release_file_content)
+        if not match:
+            from convert2rhel import redhatrelease
+            self.logger.critical("Couldn't get system version from %s" % redhatrelease.get_system_release_filepath())
+        version = namedtuple("Version", ["major", "minor"])(int(match.group(1)), int(match.group(2)))
 
-        self.logger.info("%-20s %s" % ("OS major version:", version))
+        self.logger.info("%-20s %s.%s" % ("OS version:", version.major, version.minor))
```

Comment:
Missed this the first time around

```suggestion
        self.logger.info("%-20s %d.%d" % ("OS version:", version.major, version.minor))
```